templates/model/templates/ribbon.html

```diff
@@ -0,0 +1,10 @@
+<div class="ribbon">
```

Comment:
Why do we need this file?

Reply:
To add in base.html....??

Apparently jinja2 doesn't allow top-level directory inheritance to prevent exploits.

Reply:
Maybe I should add it directly in the file, since a new file is not achieving anything in this directory.

Reply:
No, I meant that if `templates/ribbon.html` could be used in `templates/model/templates/base.html` then we won't be needing this file.